mlflow/server/js/src/common/constants.tsx

```diff
@@ -37,7 +37,7 @@ export const LoggingRunsDocUrl = `${DOCS_ROOT}/tracking.html#logging-data-to-run
 
 export const onboarding = 'onboarding';
 
-export const SupportPageUrl = 'https://github.com/mlflow/mlflow/issues';
+export const SupportPageUrl = 'https://github.com/mlflow/mlflow/issues/new?template=ui_bug_report_template.yaml';
```

Comment:
Hopefully the template helps prevent unclear issue reports like https://github.com/mlflow/mlflow/issues/18827